### PR TITLE
add `If 0, a random port is used.`

### PR DESCRIPTION
**Pull request by Hiroshiba**
# Summary

Added to the documentation that if 0 is specified for the port number, a random port number is used.

- https://github.com/encode/uvicorn/discussions/1982

# Checklist

- [x] I understand that this PR may be closed in case there was no previous discussion. (This doesn't apply to typos!)

### CODE REVIEW

**Comment by Kludex** [comment]
You'll need to run `./scripts/lint`. 🙏

**Comment by Kludex** [comment]
> You'll need to run `./scripts/lint`. 🙏

@Hiroshiba

**Comment by Hiroshiba** [comment]
Done!

File: docs/deployment.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ Usage: uvicorn [OPTIONS] APP
 Options:
   --host TEXT                     Bind socket to this host.  [default:
                                   127.0.0.1]
-  --port INTEGER                  Bind socket to this port.  [default: 8000]
+  --port INTEGER                  Bind socket to this port. If 0, a random
```
Comment (Kludex) [inline docs/deployment.md]: Let's use the following sentence instead, please:

"Bind socket to this port. If 0, an available port will be picked."

**Review by Kludex — CHANGES_REQUESTED** [review]

File: docs/deployment.md (hunk 1 above)
Comment (Kludex) [reply docs/deployment.md]: It's not random per se. 😅

Comment (Hiroshiba) [reply docs/deployment.md]: @Kludex I agree with your suggestion and have made the necessary changes. Thank you for your assistance.

**Review by Kludex — APPROVED** [review]

**Comment by Kludex** [comment]
Thanks

**Pull request merged**